content/guides/end-to-end-testing/testing-your-app.md

```diff
@@ -410,111 +410,87 @@ You'll likely also want to test your login UI for:
 
```

Comment:
The `Step 2: Visit your server` section is out of date with how Cypress works. Prob out of scope for this PR but we need to get an issue logged to get it updated with how Cypress 10+ works.

Reply:
OK I've made a note for later